src/main/resources/db/changelog/changes/init.sql

```diff
@@ -10,6 +10,7 @@ CREATE TABLE IF NOT EXISTS public.wallet
     algorithm     varchar(255) NOT NULL DEFAULT 'ED25519'::character varying,
     did_document  text         NOT NULL,
     created_at    timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+       key_storage_type VARCHAR(255) NOT NULL,
```

Comment:
Database migration will not work with the existing database

Reply:
this is the migration of the upstream branch. we would have to adapt accordingly and create a corresponding migration class

Reply:
This is a change in the existing change set, md5 checksum validation will failed

Reply:
the upstream branch has no migrations but the initital one. i can add a separate file, IF someone is actually operating the upstream branch version

Reply:
In Catena-X, they have active deployment of MIW and EDC (dev and test)
If they start with a fresh database then there will be no problem